test/test_helper.exs

```diff
@@ -1,8 +1,2 @@
-ExUnit.configure(exclude: [:redis_dependent])
 ExUnit.start()
-
-{:ok, files} = File.ls("./test/support")
-
-Enum.each(files, fn file ->
-  Code.require_file("support/#{file}", __DIR__)
-end)
+ExUnit.configure(exclude: [:redis_dependent, :skip])
```

Comment:
I would like to make sure we run those in CI at least?

Reply:
I think we should remove the `redis_dependent` tag entirely and assume we always need redis to run the tests

Reply:
fine by me then